Source/monster.cpp

```diff
@@ -586,6 +595,137 @@ void PlaceUniqueMonst(int uniqindex, int miniontype, int bosspacksize)
 	}
 }
 
+void PlaceDiablo(int x, int y)

```

Comment:
Is this code copy-pasted from somewhere? Would be nice to dedupe

Reply:
The code in here is copy pasted from PlaceUniqueMonst, but has been modified as to not alter Diablo's stats. I also use the x and y coordinates so the game knows where to place him. In PlaceUniqueMonst, no coordinates get provided to this function. Any ideas for a better workaround?